applications/nrf5340_audio/doc/requirements.rst

```diff
@@ -9,17 +9,15 @@ nRF5340 Audio application requirements
 
 The nRF5340 Audio applications are designed to be used only with the following hardware:
 
-.. table-from-rows:: /includes/sample_board_rows.txt
-   :header: heading
-   :rows: nrf5340_audio_dk_nrf5340
+.. table-from-sample-yaml::
```

Comment:
This does not seem to work to include nRF5340 or the preview build for docs has gotten stale...